classes/Infrastructure/Event/TwigGlobalsListener.php

```diff
@@ -59,7 +59,7 @@ public function __construct(
     public static function getSubscribedEvents()
     {
         return [
-            KernelEvents::REQUEST => ['onKernelRequest', 256],
+            KernelEvents::REQUEST => ['onKernelRequest', 128],
```

Comment:
Not sure about changing the priorities here, but I trust you on this one, @derrabus.